Habitica/src/main/java/com/habitrpg/android/habitica/models/notifications/GlobalNotification.java

```diff
@@ -0,0 +1,21 @@
+package com.habitrpg.android.habitica.models.notifications;
+
+import io.realm.RealmObject;
+import io.realm.annotations.PrimaryKey;
+
+/**
+ * Represents Habitica "Global notifications", i.e. the notifications about chat messages
+ * (in Guilds and Party), Party & Quest invitations, unallocated stat points etc.
+ *
+ * These are different from other kind of notifications, such as Push notifications and
+ * Popup notifications.
+ */
+public class GlobalNotification extends RealmObject {
```

Comment:
Not sure what would be good name for the model since there already exists a class called [Notification](https://github.com/HabitRPG/habitica-android/blob/1.6.1/Habitica/src/main/java/com/habitrpg/android/habitica/models/Notification.java#L12). Named it `GlobalNotification` for now but open for better suggestions.

Reply:
GlobalNotification sounds good to me for now. Do you have a plan yet for how to get the different notification types into realm? Since the data will come from multiple sources (Some of them are already stored through the user object) and with storing it in realm, you then also have to take care of removing old data. Could this maybe also be implemented as more of a viewmodel? Subscribing to p.e. the user object and then dynamically creating these notification objects from the data stored there using RxJava should work pretty well. (Please take this as a suggestion and not a demand. if you already have a plan that you think works good/better, that's good too!)

Reply:
Thanks for the suggestion, I'll take a look into it. Don't have a concrete plan yet but I've been looking into realm and was also thinking it might not be best way. Just wanted to get something working for now, so that I can get something visible to show up on screen :smiley: